platform/android/java/lib/src/org/godotengine/godot/GodotIO.java

```diff
@@ -287,6 +294,21 @@ public int getScreenOrientation() {
 		}
 	}
 
+	public int getCurrentScreenRotation() {
```

Comment:
I renamed it to `getInternalCurrentScreenRotation()` instead because it mimics what we did for `DisplayServer::screen_get_internal_current_rotation` (since it's literally the reason the Java version was added).